src/NServiceBus.NHibernate.AcceptanceTests/NServiceBusAcceptanceTest.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    public partial class NServiceBusAcceptanceTest
+    {
+        static NServiceBusAcceptanceTest()
```

Comment:
Why did you need to add this? The class as included in the source package already has this workaround. Was there some reason it needed to be done sooner?

Reply:
Huh, this is from the original PR that I based my work on. I could try to remove it @bording

Reply:
If it works without it, let's get rid of it. That also cleans up the extra bits in the other test project files.